eng/Version.Details.xml

```diff
@@ -9,384 +9,384 @@
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.Security.Cryptography.Pkcs" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.Security.Cryptography.Pkcs" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.Security.Cryptography.Xml" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.Security.Cryptography.Xml" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.Security.Permissions" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.Security.Permissions" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.Security.Principal.Windows" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.Security.Principal.Windows" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.ServiceProcess.ServiceController" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.ServiceProcess.ServiceController" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.Text.Encodings.Web" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.Text.Encodings.Web" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="System.Threading.Channels" Version="4.6.0-preview4.19157.3">
+    <Dependency Name="System.Threading.Channels" Version="4.6.0-preview4.19162.3" CoherentParentDependency="Microsoft.NETCore.App">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>00000</Sha>
+      <Sha>ef145001b10660be940aeb9318e0ff7e306f65bc</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.0.0-preview4-27507-10">
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.0.0-preview4-27512-06" CoherentParentDependency="Microsoft.NETCore.App">
```

Comment:
@mmitche as in aspnet/EntityFrameworkCore#14999, does this `CoherentParentDependency` attribute hurt?

Reply:
Nope will be ignored until supported.